test/mocks/TestToken.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.4.24;
+
+import 'openzeppelin-solidity/contracts/token/ERC20/DetailedERC20.sol';
```

Comment:
wondering if we need DetailedERC20 or if MintableToken has everything we need for the tests?